python/hail/matrixtable.py

```diff
@@ -2725,12 +2725,21 @@ def from_rows_table(cls, table: Table) -> 'MatrixTable':
         ----------
         table : :class:`.Table`
             The table to be converted.
+        partition_key : :obj:`str` or :obj:`list` of :obj:`str`
+            Partition key(s).
```

Comment:
I think key should still be singular even when it has multiple fields. How about `Partition key field(s).`?